assets/js/directives/format-input.directive.js

```diff
@@ -12,21 +12,26 @@ function formatInput () {
 
   function link (scope, elem, attrs, ctrl) {
     let inputFormat = attrs.formatInput;
-    let language = { digit: 'x' };
+    let language = { digit: 'x', open_group: '(', close_group: ')' };
```

Comment:
Are open_group and close_group required still? Is this left over from zipcode 12345-(1211)?